.github/workflows/mac.yaml

```diff
@@ -1,6 +1,9 @@
 name: Kantra CLI on MacOS
 
-on: [push]
+on:
+  push:
+    branches:
+      - 'disabled-until-fixed-or-needed'
```

Comment:
This workflow was already there, just disabling for now since it was failing for longer time and is unrelated to TIER0 kantra CI tests.